src/app/setup-wizard/data-collection/on-site-assessment/assessment-details-form/assessment-details-form.component.html

```diff
@@ -104,7 +104,7 @@ <h5>
             *ngFor="let contact of assessment.guid | associatedContacts:'assessment': contacts;"
             (click)="openContactModal(contact)">
             <fa-icon [icon]="faUser"></fa-icon>
-            {{contact.name}}
+            {{contact.firstName}} {{contact.lastName}}
```

Comment:
Instead of having `{{contact.firstName}} {{contact.lastName}}` everywhere we should use a pipe that takes in a contact and returns a formatted name. Add the pipes to the helper-pipes module. Also would be a good place to look for an example if you need one.

Reply:
added a new pipe for display contact name